src/content/docs/guides/development-workflow.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Development Workflow
+description: Learn how to integrate Shorebird into your development workflow with automated releases and patches.
+---
+
+import RemoteCode from '~/components/RemoteCode.astro';
+import CICheckSnippet from '~/components/guides/development-workflow/CICheckSnippet.astro';
+
+This guide will walk through an opinionated development workflow with Shorebird.
+We'll cover how to automate releasing and patching with Shorebird to streamline
+the development process.
+
+## Prerequisites
+
+This guide assumes you are already familiar with Shorebird and have integrated
+it into your application. Refer to the [getting started](/) instructions for
+more information.
+
+## Overview
+
+Shorebird provides a set of tools that you can use to push over the air updates
+to your customers but it's up to you to decide how to integrate it into your
+development workflow.
+
+While there are many ways to integrate Shorebird, the following workflow is a
+common approach that we recommend:
+
+![development-workflow](https://github.com/user-attachments/assets/59eaefc4-1213-4549-b0d6-38738b065147)
+
+The illustration above, uses [trunk based
+development](https://trunkbaseddevelopment.com) and tags to trigger automated
+release and patch workflows.
+
+The workflow consists of three main phases:
+
+1. **Development**: Developers work on features and bug fixes.
+1. **Release**: When a feature is ready, a release is created and distributed to
+   customers (typically via stores).
+1. **Patch**: If a critical bug is found, a patch is created and distributed to
```

Comment:
Might just be a smaller change, rather than bug.  I think some customers want to ship with Shorebird constantly for little features.

Reply:
Yeah I was mainly trying to stay focused on the critical bug fixes angle but happy to re-frame 🤷